PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
```

Comment:
> Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

This can be changed into:

Describe the items you've added to this Pull Request.

Example:
- Added new source from [link #1](#)